docs/content/routing/entrypoints.md

```diff
@@ -51,199 +56,414 @@ They define the port which will receive the requests (whether HTTP or TCP).
+    ```bash tab="CLI"
+    ## Static configuration
+    --entryPoints.name.address=:8888
+    --entryPoints.name.transport.respondingTimeouts.idleTimeout=42
+    ```
 
-## Forwarded Header
+#### `lifeCycle`
 
-You can configure Traefik to trust the forwarded headers information (`X-Forwarded-*`)
+Controls the behavior of Traefik during the shutdown phase.
 
-??? example "Trusting Forwarded Headers from specific IPs"
+??? info "`lifeCycle.requestAcceptGraceTimeout`"
+    
+    _Optional, Default=0s_
+    
+    Duration to keep accepting requests prior to initiating the graceful termination period (as defined by the `graceTimeOut` option).
+    This option is meant to give downstream load-balancers sufficient time to take Traefik out of rotation.
+    
+    Can be provided in a format supported by [time.ParseDuration](https://golang.org/pkg/time/#ParseDuration) or as raw values (digits).
+    
+    If no units are provided, the value is parsed assuming seconds.
+    The zero duration disables the request accepting grace period, i.e., Traefik will immediately proceed to the grace period.
+    
+    ```toml tab="File (TOML)"
+    ## Static configuration
+    [entryPoints]
+      [entryPoints.name]
+        address = ":8888"
+        [entryPoints.name.transport]
+          [entryPoints.name.transport.lifeCycle]
+            requestAcceptGraceTimeout = 42
+    ```
+    
+    ```yaml tab="File (YAML)"
+    ## Static configuration
+    entryPoints:
+      name:
+        address: ":8888"
+        transport:
+          lifeCycle:
+            requestAcceptGraceTimeout: 42
+    ```
+    
+    ```bash tab="CLI"
+    ## Static configuration
+    --entryPoints.name.address=:8888
+    --entryPoints.name.transport.lifeCycle.requestAcceptGraceTimeout=42
+    ```
 
+??? info "`lifeCycle.graceTimeOut`"
+    
+    _Optional, Default=10s_
+    
+    Duration to give active requests a chance to finish before Traefik stops.
+    
+    Can be provided in a format supported by [time.ParseDuration](https://golang.org/pkg/time/#ParseDuration) or as raw values (digits).
+    
+    If no units are provided, the value is parsed assuming seconds.
+    
+    !!! warning "In this time frame no new requests are accepted."
+    
     ```toml tab="File (TOML)"
+    ## Static configuration
+    [entryPoints]
+      [entryPoints.name]
+        address = ":8888"
+        [entryPoints.name.transport]
+          [entryPoints.name.transport.lifeCycle]
+            graceTimeOut = 42
+    ```
+    
+    ```yaml tab="File (YAML)"
+    ## Static configuration
+    entryPoints:
+      name:
+        address: ":8888"
+        transport:
+          lifeCycle:
+            graceTimeOut: 42
+    ```
+    
+    ```bash tab="CLI"
+    ## Static configuration
+    --entryPoints.name.address=:8888
+    --entryPoints.name.transport.lifeCycle.graceTimeOut=42
+    ```
+
+### ProxyProtocol
+
+Traefik supports [ProxyProtocol](https://www.haproxy.org/download/2.0/doc/proxy-protocol.txt) version 1 and 2.
```

Comment:
I'm not sure if ProxyProtocol works with TCP routers/services (e.g. "can I enable ProxyProtocol for a MySQL server behind traefik?").

- If it is not, we have to add a "warning" admonition here.
- If it is supported, then this comment can be dismissed.

Reply:
out of the scope of my PR for now.